test/external/clone.js

```diff
@@ -89,7 +94,7 @@ test('test github repos that use `standard`', t => {
         }
 
         function gitClone (cb) {
-          const args = ['clone', '--depth', 1, url, path.join(TMP, name)]
+          const args = ['clone', '--depth', 1, url, folder]
```

Comment:
Does this `URL` needs to be turned into a path?

```suggestion
          const args = ['clone', '--depth', 1, url, fileURLToPath(folder)]
```

Reply:
yea, folder where used on more places with child process, so i just casted the variable to path instead